### PR TITLE
File Added : JavaScript Assignments: 3,4,5,6.

### PR DESCRIPTION
**Pull request by vijaya1811**

### CODE REVIEW

File: vijaya/JavaScript/assignment4/upper.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            }
+        document.getElementById("result").value = output;
+    }
+    </script>
+    <style>
+        .name {
+            position: relative;
+            width: 500px;
+            background-color: rgba(235, 176, 66, 0.918);
+            padding: 10px 0px 10px 0px;
```
Comment (shubhamdalal49) [inline vijaya/JavaScript/assignment4/upper.html]: give proper indentation

File: vijaya/JavaScript/assignment4/upper.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <div class = "name">
+        <h3>Convert Uppercase and Lowercase </h3>
+<input type="text" id="data" placeholder="Enter text to convert"/> 
+<br/><br/>
+ <input type="button" value="Convert" onClick="convert();" style="background-color: rgb(243, 40, 50); height: 30px; color: white;"/> 
+<br/><br/>
+<input type="text"  id="result" placeholder="Result"/> 
+</body>
+</html>
+
```
Comment (shubhamdalal49) [inline vijaya/JavaScript/assignment4/upper.html]: check indentation and remove extra line from the end

File: vijaya/JavaScript/assignment5/flyoid.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                for(j=1; j<=i; j++){
+                    document.getElementById("answer").innerHTML += " "+number;
+                    number++;
+                }
+                document.getElementById("answer").innerHTML +="<br>";
+            }
+        }
+        </script>
+    </head>
+        <body style="background-color: rgb(211, 173, 92);">
```
Comment (shubhamdalal49) [inline vijaya/JavaScript/assignment5/flyoid.html:44]: check indentation

File: vijaya/JavaScript/assignment6/question2.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            }
+        </style>
+    </head>
+    <body>
+        <h1>Display Separated list, Ascending, Descending</h1>
+        <div class="one">
+            <div>
+                <p><span id="num1"></span><span id="num2"></span><span id="num3"></span></p>
+            </div>
+
```
Comment (shubhamdalal49) [inline vijaya/JavaScript/assignment6/question2.html]: remove this extra line

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

**Comment by vijaya1811** [comment]
yes I modified and also push assignment9

**Review by shubhamdalal49 — APPROVED** [review]

**Review by urmilagagare — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Pull request merged**